Automattic/wp-calypso
# Add Gravatar profile link on domain setting page

## Proposed Changes

This PR adds a link to Gravatar profile page in domain settings page, for Gravatar domains.

## Testing Instructions

Apply this patch and visit the setting page of a Gravatar domain. Verify that the Gravatar profile link is working (see screenshot below).

![gravatar profile link](https://github.com/Automattic/wp-calypso/assets/2797601/72961c6a-e2f5-4cc4-8711-0c21dc973b46)


## Pre-merge Checklist
